ci-operator/step-registry/telcov10n/functional/compute-nrop/nrop-gather-container/telcov10n-functional-compute-nrop-nrop-gather-container-commands.sh

```
@@ -0,0 +1,57 @@
+
+echo "Checking if the job should be skipped..."
+if [ -f "${SHARED_DIR}/skip.txt" ]; then
+  echo "Detected skip.txt file — skipping the job"
+  exit 0
+fi
+
+echo "Copy inventory files"
+mkdir -pv ${ECO_CI_CD_INVENTORY_PATH}/group_vars
+mkdir -pv ${ECO_CI_CD_INVENTORY_PATH}/host_vars
+
+for file in ${SHARED_DIR}/*; do 
```

Shellcheck (0.11.0)

[info] Quote expansions in this for loop glob to prevent wordsplitting, e.g. "$dir"/*.txt .

(SC2231)